junixsocket-common/src/main/java/org/newsclub/net/unix/NativeUnixSocket.java

```diff
@@ -20,25 +20,61 @@
 import java.io.FileDescriptor;
 import java.io.IOException;
 import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
 import java.net.InetSocketAddress;
-
+import java.net.Socket;
+import java.net.ServerSocket;
 /**
  * JNI connector to native JNI C code.
  * 
  * @author Christian Kohlschütter
  */
 final class NativeUnixSocket {
+  private static native void registerNatives();
   private static boolean loaded = false;
+  private static Method getMethod(Class<?> klass, String name, Class<?>...parameterTypes) {
```

Comment:
"class" is a keyword in C++.
On Jan 4, 2016 4:34 PM, "Quanah Gibson-Mount" <notifications@github.com>
wrote:

> In
> junixsocket-common/src/main/java/org/newsclub/net/unix/NativeUnixSocket.java
> <https://github.com/kohlschutter/junixsocket/pull/22#discussion_r48784215>
> :
>
> >    private static boolean loaded = false;
> > +  private static Method getMethod(Class<?> klass, String name, Class<?>...parameterTypes) {
>
> I'm not sure if junixsocket has a specific language policy. If so, it may
> be that the English "class" is preferred over the German "klass". Not that
> it matters much compile wise. ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kohlschutter/junixsocket/pull/22/files#r48784215>.
>